Dockerfile

```diff
@@ -183,3 +183,6 @@ ENV PATH=/code/venv/bin:$PATH
 WORKDIR /code
 
 CMD ["python", "/code/manage.py", "runserver", "0.0.0.0:8000"]
+
+# Reload workers after the specified amount of managed requests (avoid memory leaks)
+ENV UWSGI_MAX_REQUESTS=1000
```

Comment:
Should we set this earlier on in the file, before we start uWSGI?

Reply:
Ah yes, I think it should be up by line 89

Reply:
I moved it up. Great suggestion @dchukhin. I moved it 🆙.